### DISCUSSION TITLE
Possible Security Risk - unauthorized admin access

### DISCUSSION THREAD

**Discussion by Spsrky** [code4recovery/12-step-meeting-list#986] [Q&A] [answered]
Got an email from a user indicating that they found they can get admin access to the site by using this code in the address bar: <code>https://www.pdxaa.org/wp-admin/post.php?post=38876&action=edit</code>

 This takes you to the edit page for that meeting but appears to allow admin access to the whole site dashboard. I am not able to reproduce this on a clean browser that does not have my history. Has anyone encountered this?

**Comment by kiyote33** [comment, marked as answer]
Hi @Spsrky,

I'm not able to get admin access using the posted link https://www.pdxaa.org/wp-admin/post.php?post=38876&action=edit.  I simply get redirected to the login, which is what I'd expect.

**Comment by Spsrky** [comment]
Thanks for checking that out, Wayne. The initial email we got was very brief, just had the link, like it was something we should know about. I tried it using a clean browser that didn't have my login credentials in the history and I couldn't find anything scary. I've replied to the original email to see what it was he was on about.

In fellowship and service,

Theron
